src/whitepaper/accelerating-innovation-in-manufacturing-with-flowfuse.njk

```diff
@@ -0,0 +1,21 @@
+---
+layout: layouts/base.njk
```

Comment:
The extends is nice on line 15, but why not use this line instead?

Reply:
I think if the layout is `whitepaper`, all the `blocks` below can be in the frontmatter?